types/index.d.ts

```diff
@@ -287,33 +287,42 @@ export const MTableToolbar: (props: any) => React.ReactElement<any>;
 export const MTable: (props: any) => React.ReactElement<any>;
 
 export interface Icons {
```

Comment:
Can we remove these changes that are not part of the PR?

Reply:
Yes done. Sorry :)